source/clear-linux/tutorials/multi-boot/multi-boot-rhel.rst

```diff
@@ -134,15 +132,15 @@ Install Red Hat\* Enterprise Linux 7.4 Beta
 
       .. note:: The root partition UUID used below is unique to this example.
 
-         .. code-block:: console
+         .. code-block:: bash
 
-            # cd /boot/efi/loader/entries
+            sudo cd /boot/efi/loader/entries
 
-            # vi redhat.conf
+            sudo vi redhat.conf
```

Comment:
Let's consolidate lines 138 and 139 to "sudoedit /boot/efi/loader/entries/redhat.com"

Reply:
Changed to "sudoedit /boot/efi/loader/entries/redhat.CONF" to match original file type, OK?